build/docker/copy-test-operations.sh

```diff
@@ -19,5 +19,6 @@ mkdir -p $USERLIB
 echo "Copying test artifacts to install location..."
 find ${WORKSPACE}/conformance -iname 'fhir-ig*.jar' -not -iname 'fhir*-tests.jar' -not -iname 'fhir*-test-*.jar' -exec cp -f {} ${USERLIB} \;
 cp -pr ${WORKSPACE}/operation/fhir-operation-test/target/fhir-operation-*-tests.jar ${USERLIB}
+cp -pr ${WORKSPACE}/operation/fhir-operation-everything/target/fhir-operation-*.jar ${USERLIB}
```

Comment:
now the 4.6.1 is shipped, I'm hoping that we can just put the operation right into our webapp (like all the others).

Reply:
Removed the provided and changed the script back